AEM prerender docs for developers and merchants (#543)
commerce-docs/microsite-commerce-storefront: src/content/docs/merchants/get-started/prerendered-product-pages.mdx

---
title: Prerendered product pages
description: Learn how prerendered product pages improve your store's performance and search engine visibility.
---

import { LinkCard, Steps } from '@astrojs/starlight/components';
import Diagram from '@components/Diagram.astro';
import Aside from '@components/Aside.astro';

Prerendered product pages are generated and published before customers visit them. The system creates and stores each page in advance, delivering it instantly when requested.

<Aside type="note" title="Developer Setup Required">

> Your development team configures the prerender system. Once set up, it runs automatically with no changes to your workflow.

Reviewer: ```suggestion
Your development team configures the prerender system. Once set up, it runs automatically without changing your workflow.
```

</Aside>

## What you need to do

**Nothing different.** Manage your products in Adobe Commerce as usual. The prerender system works automatically in the background.

<Diagram 
  type="mermaid" 

>   caption="How prerendering works: you update products, the system generates pages automatically, and customers get fast, SEO-optimized pages."

Reviewer: ```suggestion
  caption="How prerendering works: You update products, the system generates pages automatically, and customers get fast, SEO-optimized pages."
```

    C["<span style='font-size:80px'>⚡</span><br/><span style='font-size:18px'>Pages Delivered</span>"]:::delivery
    D["<span style='font-size:80px'>🛒</span><br/><span style='font-size:18px'>Happy Customers</span>"]:::customer
    
    A -->|"In Adobe Commerce<br/>(business as usual)"| B
    B -->|"Automatically detects<br/>changes & creates pages"| C
    C -->|"Instantly to customers<br/>& search engines"| D
    
    classDef merchant fill:none,stroke:none,color:#000000
    classDef auto fill:none,stroke:none,color:#000000
    classDef delivery fill:none,stroke:none,color:#000000
    classDef customer fill:none,stroke:none,color:#000000
`}
/>

## Why it matters for your store

Prerendered pages improve your store's performance and visibility:

### Better search rankings

Search engines index your product information immediately. Your products appear in search results faster, helping shoppers find your store.

### Faster page loads

Pages load instantly, improving:

- Customer retention
- Product views
- Conversion rates
- Mobile experience

**Performance impact:** Most product pages load in under one second.

### AI and social media ready

> Product links on social media display your images, descriptions, and pricing automatically. AI shopping assistants can understand and recommend your products.

Reviewer: ```suggestion
- **Better search rankings:** Search engines index your product information immediately. Your products appear in search results faster, helping shoppers find your store.
- **Faster page loads:** Pages load instantly, improving:
  - Customer retention
  - Product views
  - Conversion rates
  - Mobile experience
- **Performance impact:** Most product pages load in under one second.
- **AI and social media ready:**  Product links on social media display your images, descriptions, and pricing automatically. AI shopping assistants can understand and recommend your products.
```

## How the system works

<Steps>

1. You update products in Adobe Commerce
2. The system detects changes automatically
3. Product pages generate and publish in the background

> 4. Customers see updated content within minutes

Reviewer: ```suggestion
1. You update products in Adobe Commerce.
2. The system detects changes automatically.
3. Product pages generate and publish in the background.
4. Customers see updated content within minutes.
```
